internal/utils/flags.go

```diff
@@ -0,0 +1,46 @@
+package utils
+
+import (
+	"strings"
+
+	ma "github.com/multiformats/go-multiaddr"
+)
+
+// AddrList is a list of multiaddress
+type AddrList []ma.Multiaddr
+
+// String is a function to print a string representation of the AddrList
+func (al *AddrList) String() string {
+	strs := make([]string, len(*al))
+	for i, addr := range *al {
+		strs[i] = addr.String()
+	}
+	return strings.Join(strs, ",")
+}
+
+// Set is a function to set the value of AddrList based on a string
+func (al *AddrList) Set(value string) error {
```

Comment:
Given that it's a list, and this func is appending. Better to name it Add or Append

Reply:
I has to be named set as it implements the golang flag interface.
https://golang.org/pkg/flag/#pkg-examples